apps/omg_performance/lib/performance.ex

```diff
@@ -158,6 +158,7 @@ defmodule OMG.Performance do
 
     Application.put_env(:omg_eth, :contract_addr, contract_addr)
 
+    # FIXME
```

Comment:
Somehow this line does not trigger credo, while it should.

Reply:
apparently `# FIXME a` works, I'll file an issue to `credo`